tests/test_oauth2client.py

```diff
@@ -144,6 +146,22 @@ def get(self, key, default=None):
     return Info(self._headers)
 
 
+@contextlib.contextmanager
+def mock_module_import(module):
+  """Place a dummy object in sys.modules to mock an import test"""
+  parts = module.split('.')
+  entries = ['.'.join(parts[:i+1]) for i in range(len(parts))]
+  for n in entries:
```

Comment:
Please don't use "n" for values that aren't integers.

I think "entry" or "fully_qualified_module_name" are both fine names for this value.